src/components/Menus/FileMenu/FileMenu.test.js

```diff
@@ -1,76 +1,70 @@
-import React from "react";
-import { render } from "@testing-library/react"
-import { Provider } from 'react-redux';
-import configureStore from 'redux-mock-store';
-import { MemoryRouter } from "react-router-dom";
+import { fireEvent, getByText, render } from '@testing-library/react'
```

Comment:
There's a linting error here:
```
'getByText' is defined but never used
```
which is odd, because it appears to be used.

Reply:
as in the IDE imported it for me as it matched a symbol in an import library